Automattic/wp-calypso
# Pricing Grid: Update breakpoints

#### Proposed Changes

* This PR updates the breakpoints for mobile, tablet, and desktop variations as per the interpretation shared [here](https://github.com/Automattic/wp-calypso/pull/71532#pullrequestreview-1232474153). The previous implementation used the logic described in p1672323567403859-slack-C040UAN6TPS.

#### Testing Instructions



* Verify that the price grid width is appropriately rendered for the breakpoints as mentioned [here](https://github.com/Automattic/wp-calypso/pull/71532#pullrequestreview-1232474153).
